src/astro/sql/operators/sql_decorator.py

```diff
@@ -94,23 +95,24 @@ def execute(self, context: Dict):
         self.run_id = context.get("run_id")
         self.convert_op_arg_dataframes()
         self.convert_op_kwarg_dataframes()
-        if not self.sql:
+        if self.sql is None or self.sql == "":
```

Comment:
@utkarsharma2 it may be worth adding a comment why we are not doing `not self.sql` - people less familiar with SQLAlchemy might be tempted to write as it was before